package/chalicelib/reportGenerator.py

```diff
@@ -42,6 +45,30 @@ def __init__(self, start_date, end_date, granularity='DAILY', metrics=None):
         self.nums_to_aliases, self.aliases_to_nums = self.build_nums_to_aliases_dicts()
         self.account_nums = list(self.nums_to_aliases.keys())
 
+        credentials = self.get_email_credentials(secret_name)
+        self.email = list(credentials.keys())[0]
+        self.password = credentials[self.email]
+
+    @staticmethod
+    def get_email_credentials(self, secret_name):
+        """
+        Retrieve email address and password pair from AWS Secrets Manager
+
+        :param str secret_name: The id of the secret in AWS. Can be ARN or friendly name
+        :return: dict in the format {you@gmail.com: p@ssw0rd}
+        """
+        region_name = "us-west-2"
+
+        # Create a Secrets Manager client
+        session = boto3.session.Session()
+        client = session.client(service_name='secretsmanager', region_name=region_name)
```

Comment:
I think there is a cleaner way to create this client:
      client = boto3.client('secretsmanager', region_name=region_name)